src/applications/claims-status/sass/claims-status.scss

```diff
@@ -135,6 +151,42 @@
   }
 }
 
+.submitted-on,
+h3 .usa-label {
+  display: block;
+  width: fit-content;
+  font-size: 1.6rem;
+  font-family: var(--font-source-sans);
+  font-style: normal;
+  font-weight: normal;
+  line-height: 1.5;
+  margin-bottom: 0.5em;
+  margin-top: 0.5em;
```

Comment:
I think it's safe to remove these. It seems like it makes more sense to put the margin values in the specific selectors (i.e. `.submitted-on` and `h3 .usa-label`)